scripts/run-integration-tests.sh

```diff
@@ -35,10 +35,9 @@ create-eth-keys() {
 		echo "${GANACHE_KEYS[${i}]}" >"${key_file}"
 	done
 }
-
 # This is the local multiaddr created when using ./tests/alice-libp2p.key on the default libp2p port
-ALICE_MULTIADDR=/ip4/127.0.0.1/udp/9933/quic-v1/p2p/12D3KooWAAxG7eTEHr2uBVw3BDMxYsxyqfKvj3qqqpRGtTfuzTuH
-ALICE_LIBP2PKEY=./tests/alice-libp2p.key
```

Comment:
The variable name and key were renamed to "relayer", but it is referenced 3 times as Alice's key and only once as the relayer's key. Would it make sense to use separate keys?

Reply:
@dimalinux let me know if you wanna take a stab at the script like we discussed

the new flow would be:

- Alice starts first, deploys the forwarder contract and then swap contract which references that
- Bob/Charlie/relayer can all start after, Bob/Charlie use the swap contract deployed by Alice and the relayer uses the forwarder contract deployed by Alice

I didn't update it b/c I didn't want to mess with the log parsing, but feel free to take a stab at it.